index.js

```diff
@@ -0,0 +1,141 @@
+  },
+  {
+    title: 'Becoming Best sellers',
+    image: './img/Buku 4 (1).png',
+    alt: 'Book2',
+    description: 'Becoming Best sellers is an amazing book it ...',
+  },
+  {
+    title: 'The Sielent Patient!',
+    image: './img/Buku 5 (1).png',
+    alt: 'Book2',
+    description: 'The Sielent Patient! book revolves around.... ',
+  },
+  {
+    title: 'The Lord of Rings',
+    image: './img/Buku 6.png',
+    alt: 'Book2',
+    description: 'The Lord of Rings Amazing fiction based....',
+  },
+];
+
+const HotBookSection = [
+  {
+    title: 'The Hobbits',
+    image: './img/h1.png',
+    alt: 'HotBook1',
+    description: 'Books are referred to as a mans best friend. They are very beneficial for mankind and have helped it evolve.',
+  },
+  {
+    title: 'Harry Porter',
+    image: './img/h2.png',
+    alt: 'HotBook2',
+    description: 'Books are referred to as a mans best friend. They are very beneficial for mankind and have helped it evolve.',
+  },
+  {
+    title: 'The Subtle Art',
+    image: './img/h3.png',
+    alt: 'HotBook3',
+    description: 'Books are referred to as a mans best friend. They are very beneficial for mankind and have helped it evolve.',
+  },
+  {
+    title: 'Think like a monk',
+    image: './img/h4.png',
+    alt: 'HotBook4',
+    description: 'Books are referred to as a mans best friend. They are very beneficial for mankind and have helped it evolve.',
+  },
+  {
+    title: 'The Hobbits',
+    image: './img/h1.png',
+    alt: 'HotBook1',
+    description: 'Books are referred to as a mans best friend. They are very beneficial for mankind and have helped it evolve.',
+  },
+  {
+    title: 'The Subtle Art',
+    image: './img/h3.png',
+    alt: 'HotBook3',
+    description: 'Books are referred to as a mans best friend. They are very beneficial for mankind and have helped it evolve.',
+  },
+
+];
+
+const ShowBooks = () => {
+  for (let i = 0; i < BookSectionOBJ.length; i += 1) {
+    BookSection += `
+      <div class="book-holder">
+      <div class="image-holder">
+          <img src="${BookSectionOBJ[i].image}" alt="${BookSectionOBJ[i].alt}" width="100">
+      </div>
+      <div class="description whitecolor">
+      <div class="booktitle primarycolor">${BookSectionOBJ[i].title}</div>
+      ${BookSectionOBJ[i].description}
+      </div>
+  </div>
+      `;
+  }
+  if (BookContainer !== null) {
+    BookContainer.innerHTML = BookSection;
+  }
+};
+
+ShowBooks();
+BookSection = '';
+
+const ShowHotBooks = () => {
+  for (let i = 0; i < HotBookSection.length; i += 1) {
+    BookSection += `
+        <div class="Hotbook-holder book-holder-color">
+        <div class="image-holder">
+            <img src="${HotBookSection[i].image}" alt="${HotBookSection[i].alt}" width="100">
+        </div>
+        <div class="description">
+        <div class="booktitle primarycolor">${HotBookSection[i].title}</div>
+        ${HotBookSection[i].description}
+        </div>
```

Comment:
- [ Optional ] I would suggest left aligning the book titles can improve your layout. Besides,  how about we give some gap between the Book Section tile and its contents?
![speakers-section-impl](https://user-images.githubusercontent.com/44057863/169401006-c20827cd-0095-42ab-8027-d0507de4e4a9.png)

Reply:
I'll definitely consider it